### ISSUE TITLE
Getting 'no command line launcher' error, when I do have it

### ISSUE THREAD

**Issue by ilyador** [bchatard/jetbrains-alfred-workflow#3]
Trying to use it with WebStorm 11.0.3, I have /usr/local/bin/wstorm but still getting error in Alfred that it is not found.

https://www.dropbox.com/s/0z3sa9qx2peoes9/Screenshot%202016-02-25%2013.07.13.png?dl=0

**Comment by bchatard** [comment]
did you install Webstorm via brew cask? if yes try to re-create the command line tools

**Comment by ilyador** [comment]
No,
And tried to re-create anyway but still the same error

**Comment by bchatard** [comment]
can you provide me this information:
- Zsh version: `/bin/zsh --version`
- grep version: `grep --version`
- grep location: `whereis grep`
- result of this command: `grep -F -m 1 'RUN_PATH =' /usr/local/bin/wstorm`

**Comment by ilyador** [comment]
```
zsh 5.0.8 (x86_64-apple-darwin15.0)
grep (BSD grep) 2.5.1-FreeBSD
/usr/bin/grep
RUN_PATH = u'/Applications/WebStorm.app'
```

**Comment by bchatard** [comment]
are you on OSX10.11?
can you run following command: `ls -hAl /Applications/WebStorm.app`

**Comment by ilyador** [comment]
Yes

```
$ ls -hAl /Applications/WebStorm.app
total 0
drwxr-xr-x  12 user  admin   408B Dec 24 14:55 Contents
```

**Comment by bchatard** [comment]
I'm not yet on OSX10.11...
I'll get a Mac on this version next week (I hope), to debug this

**Comment by bchatard** [comment]
@ilyador ok I get the bug on OSX10.11, I will fix this this week-end

**Comment by bchatard** [comment]
@ilyador can you try the version available on MR #4
